cirq-core/cirq/contrib/paulistring/convert_to_clifford_gates.py

```diff
@@ -12,18 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional, TYPE_CHECKING
+from typing import Optional
 
-import numpy as np
-
-from cirq import ops, protocols, transformers, linalg
+from cirq.contrib.paulistring.clifford_target_gateset import _matrix_to_clifford_op
```

Comment:
Is there any difference in behavior w/r to the `_matrix_to_clifford_op` method deleted at the original line 66?

If not it is perhaps better to keep the deleted methods as they were and avoid importing private function from another module.  The class is deprecated so the duplicate code will be removed later together with the class.

Reply:
The file has been deleted and only a single implementation of `_matrix_to_clifford_op` exists now; so this is moot.